TYPES.md

```diff
@@ -100,6 +103,70 @@ Parameters are :
 | secondary |
 | `hasKey(<key>)(state)` | returns `true` if the given `<key>` is present | |
 
+## simple
+### factory
+```es6
+import { simple } from 'k-redux-factory'
+
+export default simple(/* options */)
+```
+
+or
+
+```es6
+import factory from 'k-redux-factory'
+
+export default factory({ type: 'simpleObject', /* other options */ })
```

Comment:
type: 'simple' works ?
type: 'simple.bool' works ?

Reply:
Does not work at the moment.
But your purpose is : 
 - [] rename folder `simpleObject` to `simple`
 - [] rename type `simpleObject` to `simple`
 - [] add type for each shorter ?
 - [] keep on deprecated type `simpleObject`
?

Reply:
- Yes you can rename folder and imports and file
 - You can add type: 'simple.xxx' to the factory
 - You can keep simpleObject